test/spec/modules/userId_spec.js

```diff
@@ -626,11 +627,10 @@ describe('User ID', function() {
     });
 
     it('does not delay auction if there are no ids to fetch', function() {
-      coreStorage.setCookie('MOCKID', JSON.stringify({'MOCKID': '123456778'}), new Date(Date.now() + 5000).toUTCString());
-
+      coreStorage.getCookie.withArgs('MOCKID').returns('123456778');
```

Comment:
Can you elaborate this fix ? Original test is setting cookie and you updated it to get cookie. Does this change the unit test completely ? 
Also you reduced the auction delay to 33. What do we achieve by doing that ?

Reply:
@jaiminpanchal27 
I mocked `getCookie` function because at [this](https://github.com/prebid/Prebid.js/blob/42e41a0c06de915e359d60b6571acaf74374a08d/modules/userId/index.js#L196) line, `coreStorage.getCookie` is returning `undefined.` Because of this, the test case was failing half of the times.

Ideally, while unit testing a module, we should not be calling any external library or functions. So, I mocked it to return a set value.

Auction delay was previously set to `33`, in [this](https://github.com/prebid/Prebid.js/pull/4982/files) PR I changed it to `200`, but that fix didn't work. So kinda, reverting it back. Anyway, `auctionDelay` should not matter much, whatever we set it to.